src/components/CalendarMonth.jsx

```diff
@@ -25,6 +26,7 @@ const propTypes = forbidExtraProps({
   enableOutsideDays: PropTypes.bool,
   modifiers: PropTypes.object,
   orientation: ScrollableOrientationShape,
+  daySize: nonNegativeInteger,
```

Comment:
I think this does need to be nonNegativeInteger(). This leads to a lot of warnings for me, of the form:

```
Warning: CalendarDay: type specification of prop `daySize` is invalid; the type checker function must return `null` or an `Error` but returned a function. You may have forgotten to pass an argument to the type checker creator (arrayOf, instanceOf, objectOf, oneOf, oneOfType, and shape all require an argument).

```

I have opened a PR to fix this issue: https://github.com/airbnb/react-dates/pull/664